### PR TITLE
supporting `getFontScale` in App Extensions

### LINKED ISSUE #625: Update getFontScale to work in iOS App Extensions
## Summary

|             |     |
| ----------- | --- |
| Version     | master   |
| Affected OS | iOS   |
| OS Version  | any   |

## Current behavior

Following up #408, we’ve patched the library to compile with iOS App Extensions, however getFontScale always returns 1.0 since the shared application cannot be accessed. A potential (iOS 10+) workaround is described in https://github.com/react-native-community/react-native-device-info/pull/408#issuecomment-462807763.

## Expected behavior

getFontScale should probably return the font scale on iOS 10+ and fall back to 1.0 or use a different workaround with older versions.

### PR DESCRIPTION
**Pull request by schie**
## Description

Fixes #625

- updates how contentSize is obtained; using `[[UIScreen mainScreen] traitCollection];`
- updates example to include an app extension for testing and such **(MAJORITY OF THIS PR)**

BREAKING CHANGE: by obtaining the content size from `[[UIScreen mainScreen]
traitCollection].preferredContentSizeCategory` , this library drops support for iOS 9. This should
be fine since RN 0.63+ dropped support for iOS9 and the OS, itself, has <1% iOS market share.

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅     |
| Android |    ❌     |
| Windows |    ❌     |

## Checklist

* [X] I have tested this on a device/simulator for each compatible OS
* [X] I added the documentation in `README.md`
* [ ] I mentioned this change in `CHANGELOG.md`
* [ ] I updated the typings files (`privateTypes.ts`, `types.ts`)
* [X] I added a sample use of the API (`example/App.js`)

## TL;DR
image

### CODE REVIEW

**Comment by schie** [comment]
The addition of the app extension could probably have been more atomic, but it was just this 'nebulus of stuff' before getting it to work that was seemingly codependent with one another at the time.

**Comment by schie** [comment]
@mikehardy,

Is there anything I'm missing on this?

**Comment by mikehardy** [comment]
Oh dang. Just missing my attention was in a conference last 2 days and am underwater on everything else. Thanks for your patience

**Comment by schie** [comment]
No worries

**Review by mikehardy — APPROVED** [review]

**Comment by mikehardy** [comment]
finally got a second to work through my gratis libraries :), merged this in. Will pull really quickly for a test then release, thanks @schie !

**Comment by mikehardy** [comment]
shipped! appreciate the patience @schie 

https://github.com/react-native-device-info/react-native-device-info/releases/tag/v7.0.0

**Comment by schie** [comment]
@mikehardy,
not a problem at all!

Also, I just noticed that all/most of the `react-native-community` libs moved out of that organization, is there a reason for that?

**Comment by mikehardy** [comment]
> @mikehardy,
> not a problem at all!
> 
> Also, I just noticed that all/most of the `react-native-community` libs moved out of that organization, is there a reason for that?

Progress? Organization re-organization

https://github.com/react-native-community/discussions-and-proposals/blob/master/partners/0001-organization-repository-policy.md

🤷 my take away is that coordinating a multi-organization dev effort in the open is not easy
